pkg/consuldp/bootstrap.go

```diff
@@ -76,6 +77,11 @@ func (cdp *ConsulDataplane) bootstrapConfig(ctx context.Context) (*bootstrap.Boo
 		PrometheusScrapePath:  prom.ScrapePath,
 	}
 
+	if rsp.ClusterName != "" {
```

Comment:
Same comment about naming from the Consul PR if this is supposed to be `identity`.

Reply:
Changed it to identity.